.github/workflows/release-porting-guide.yml

```diff
@@ -0,0 +1,103 @@
+---
+name: Ansible porting guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-build-data-branch:
+        description: >-
+          Release Branch name from the Ansible Build data PR.
+          (e.g. `refs/pull/1/merge`)
+        required: true
+      ansible-version:
+        description: >-
+          Exact release version. For exmaple, 12.1.0
```

Comment:
```suggestion
          Exact release version. For example, 12.1.0
```

Reply:
Duplicate of https://github.com/ansible/ansible-documentation/pull/121/files#r2192823293